src/main/java/pro/sky/telegrambot/model/NotificationTask.java

```diff
@@ -0,0 +1,84 @@
+package pro.sky.telegrambot.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+@Entity
+public class NotificationTask {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private Long id_chat;
```

Comment:
```suggestion
    private Long idChat;
```
camelCase, а не snake_case